src/db/migrations/3_CreateConfigTable.js

```diff
@@ -0,0 +1,31 @@
+const debug = require("../debug");
+
+class CreateConfigTable extends require("../Migration") {
+  async body(index, database) {
+    let done = false;
+    let sql = await this.sql();
+
+    // await sql.schema.dropTableIfExists("test");
+
+    if (!(await sql.schema.hasTable("scraper_config"))) {
+      await sql.schema.createTable("scraper_config", async (table) => {
+        table.increments("id").primary();
+        table.string("name").notNullable().unique();
+        table.string("address").notNullable().unique();
+        table.integer("chain_id").notNullable();
+        table.integer("start_block").notNullable();
+        table.bool("started");
+        table.json("events");
+        table.timestamp("created_at").defaultTo(sql.fn.now());
```

Comment:
We should add a version for the contract so that instead of renaming it, we just add the version.
The version can be an incremental integer. Sometimes, we may want to reset the table, in that case, it makes sense not to make the address unique. Just keep the name unique because we will search by name.

Reply:
Make the version a string and default it to "" so that all the existing tables without a version will be uneffected, while the new one will use it. Let's say we have
PFPAsset (no version)
and
PFPAsset (version "2")

The function that define the table should change to:
```
  nameTable(contractName, filter, version) {
    let attrs = filter
      .split("(")[1]
      .split(")")[0]
      .split(",")
      .map((e) => e.substring(0, 1));
    return [
      Case.snake(contractName),
      version || "",
      Case.snake(filter.split("(")[0]),
      attrs.join(""),
    ].join("__");
  },
  ```
  so the first table name will be
  ```
  pfpasset__transfer__aau
  ```
  and the second
  ```
  pfpasset__2__transfer__aau
```